internal/api/handlers/collectibles.go

```diff
@@ -243,6 +246,13 @@ func (h *CollectiblesHandler) GetCollectibles(w http.ResponseWriter, r *http.Req
 	ctx, cancel := context.WithTimeout(r.Context(), HealthCheckContextTimeout)
 	defer cancel()
 
+	network := r.URL.Query().Get("network")
+	if network != types.PUBLIC && network != types.TESTNET && network != types.FUTURENET {
```

Comment:
In order to support Freighter Mobile which is already using this endpoint in prod, we won't throw an error on a missing network config yet. We'll default to Pubnet to maintain the current functionality